washingtonpost/elex-live-model
# elex-2453-called-states-base-values-TEMP PR

## Description
This is just a PR to make it easier for you to see what I've changed in my model branch. Let's just delete this PR after your check. 

We need to incorporate states that have already been called in the aggregate model. We also need to account for “base values” or the seats already assigned pre -election (this applies to the senate, not president). For example if only 30 senate seats are up for election, we want to store in the model how the remaining 70 are already distributed, so that our final agg plots show estimates for the TOTAL senate seats predicted for each party.

Note, new input files are added for called-states and base-values, but these will of course eventually come from the database. The first will be data that is updated throughout the night and the second is of course static.

You need to be on my model branch (agg-dev-contd-with-bug-fix) and my testbed branch (new-corr-data).

If this structure generally looks good, I can open a testbed PR.

**Main pieces added:**
For called states: BaseElectionModel.py lines 333-340 (and related data read-ins)
For base-values: BaseElectionModel.py lines 421-426 (and related data read-ins) 
New input files

Note: not all merge tests have passed because of the elex-static license issue that I'm still working out. But that is fine for our purposes rn.

## Jira Ticket
https://arcpublishing.atlassian.net/jira/software/c/projects/ELEX/boards/1026?modal=detail&selectedIssue=ELEX-2453

## Test Steps
1. For both President 2020 and senate 2022 play with values in new input files (no base values for 2020 pres) and make sure values are reflected in total agg outcomes.

2. Sanity check where I’ve incorporated both in the model make sense for each.
